src/analysis/modals/ComputeModal/AzureComputeModal/AzureComputeModal.test.ts

```diff
@@ -221,6 +230,7 @@ describe('AzureComputeModal', () => {
     const numberInput = await screen.getByLabelText('minutes of inactivity');
     expect(numberInput).toBeInTheDocument();
     await user.type(numberInput, '0');
+    // @ts-expect-error
```

Comment:
ts-expect-error is bad because any error might sneak in beyond the one this is attended to silence.  It is better to fix the underlying problem... what is the ts error you are seeing?

if all else fails, it's better to cast the problem as any then to take on the risks of ts-expect-error.

Reply:
A simple coercion of the return to a `HTMLInputElement` does it here